content/guides/integration-guides/csi-integration/soda-csi-pnp/_index.md

```diff
@@ -0,0 +1,75 @@
+---
+title: SODA Integration with Vendor CSI Drivers via SODA CSI Plug-N-Play
+menuTitle: "SODA CSI Plug-N-Play"
+description: "A guide for integrating SODA with Vendor CSI Driver via unified Plugin."
+weight: 10
+disableToc: false
+tags: ["integration guide", "kubernetes csi", "plug-n-play", "unified plugin"] 
+---
+
+SODA CSI Plug-N-Play is a mechanism by which users can use the heterogeneous storage vendors in a unified way, 
+Users need to define the requirements while creating SODA Profile, 
+Users need to use this profile ID while creating the PVC's in the Kubernetes, 
+the soda-csi-provisioner will automatically select the vendor csi-driver and help in 
+provisioning the Volumes for Pods.
+  
+## Prerequisite
+ - An installation of Kubernetes (V1.17+)  
+ - SODA installation (you can refer to quick start guide over [here](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)), you need to change the ports of etcd in osdb.yaml while installing SODA on an existing K8s environment as the ports cause conflict with k8s etcd.
+
+## Selecting the vendor CSI driver
+You can select the CSI driver supported by SODA Plug-N-Play from [here](https://docs.sodafoundation.io/guides/user-guides/nbp/csi-pnp/) Once selected follow the instruction 
+given in the document to deploy the CSI driver in K8s.  
+
+##### For this example we are selecting the [OpenEBS LVM CSI Driver](https://github.com/openebs/lvm-localpv).
+
+```go
+kubectl create -f https://raw.githubusercontent.com/asifdxtreme/soda-ucp/main/examples/openebs/driver/lvm-operator.yaml
```

Comment:
though it's an example, can we give sodafoundation project reference

Reply:
will move this example https://github.com/sodafoundation/examples/issues/11